nodewatcher/core/frontend/urls.py

```diff
@@ -12,4 +14,6 @@
     # Frontend components auto-discovery
     components.pool.discover_components()
 
-    urlpatterns = components.pool.get_urls()
+    urlpatterns = components.pool.get_urls() + [
+        urls.url(r'^api/v2/', urls.include(api.router.urls)),
```

Comment:
So to me frontend is this particular Django component system we have and display stuff.